portfolio/src/main/webapp/index.html

```diff
@@ -110,13 +94,24 @@ <h2>
         <div class="desc">Tokyo, Japan</div>
       </div>
 
-      <p>I would love to travel to these cities (and many more) to experience its culture. I personally believe that my life would not be complete withoug immersing myself in places outside my comfort zone. In addition to my love for traveling, I also enjoy taking photos of people, cities, and nature. Capturing a landscape is one of the most beautiful things. Capturing a busy city is even more captivating. I spent my previous summer in New York City, and some of the photos of the city are some of my favourites.</p>
+      <h3>Why?</h3>
+      <p>I would love to travel to these cities (and many more) to experience its culture. I personally believe that my life would not be complete withoug immersing myself in places that I have never been before. In addition to my love for traveling, I also enjoy taking photos of people, cities, and nature. Capturing a landscape is one of the most beautiful things. Capturing a busy city is even more captivating. I spent my previous summer in New York City, and some of the photos of the city are some of my favourites.</p>
       <hr>
-      <h3>
-        <cursive>My Hidden Talent</cursive>
-      </h3>
+      <h3>My Hidden Talent</h3>
       <p>Although I write with my left hand, I am actually ambidextrous. I do many activities with both hands and can even write with both hands (not very well, but its legible)!</p>
-      <p><cursive>Connect with me on <a href=https://www.linkedin.com/in/sydney-beasley-6b04b8168>LinkedIn</a> and <a href=https://www.github.com/Sydney-18>GitHub</a></cursive></p>  
+      <hr>
+      <h2>Social Media</h2>
+
+      <head>  
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+      </head>  
+        
+      <a href="https://www.linkedin.com/in/sydney-beasley-6b04b8168" class="fa fa-linkedin"></a>
```

Comment:
Where is the visual piece of these links coming from?

Reply:
These allow me to use social media icons on my site.

Reply:
Are they being added in already somewhere? Is that part of the font-awesome stylesheet you added in?

Reply:
The icons are only added in at this one spot and, yes, they are part of the font-awesome stylesheet.

I'm not sure if you are able to see the screenshot but here's the link:
https://screenshot.googleplex.com/H0b68YUJzBq

Reply:
Excellent, thanks for clarifying that. I would suggest that you add an HTML comment above the icons section that says something like `<!-- Social icons populated by CSS. -->` so that people who might be reading through your code later don't have to search around to understand how things are working. Nice for you, too, when you've been away from the code for a year and have to remember how everything works. :)

Reply:
Gotcha, thanks!